input/blog/2021-02-07-cake-v1.0.0-released.md

```diff
@@ -0,0 +1,319 @@
+
+```bash
+The assembly 'Cake.DotNetTool.Module, Version=0.4.0.0, Culture=neutral, PublicKeyToken=null'
+is referencing an older version of Cake.Core (0.33.0).
+```
+
+### Rewritten CLI
+
+We're now utilizing [Spectre.Cli](https://spectresystems.github.io/spectre.cli/) for a more robust, flexible and future proof handling of command line input.
+
+Full details of everything that was included in this release can be seen below.
+
+### Frosting incorporated into main repository
+
+Cake Frosting is now incorporated in the same GitHub repository as ( [cake-build/cake](https://github.com/cake-build/cake/) ) all the other Cake runners, this means it'll be released and supported with the same cadence as them.
+
+### Breaking changes in Cake Frosting
+
+In this release the `UseTool` extension has been renamed to `InstallTool`.
+In our prior release candidate we also [noted](/blog/2020/12/cake-v1.0.0-rc0002-released#breaking-changes-in-cake-frosting) that `CakeHostBuilder` has been removed and `CakeHost` can be used directly.
+
+With Cake.Frosting 0.38.x:
+
+```csharp
+// Create the host.
+var host =
+    new CakeHostBuilder()
+        .WithArguments(args)
+        .UseStartup<Program>()
+        .UseTool(new Uri("nuget:?package=ReportGenerator&version=4.8.1"))
+        .Build();
+
+// Run the host.
+return host.Run();
+```
+
+With Cake.Frosting 1.0:
+
+```csharp
+// Create and run the host.
+return
+    new CakeHost()
+        .UseContext<BuildContext>()
+        .InstallTool(new Uri("nuget:?package=ReportGenerator&version=4.8.1"))
+        .Run(args);
+```
+
+### Supported operating systems
+
+The following table shows the supported operating systems for each runner.
+
+| Runner                           | Windows | macOS | Linux |
+|----------------------------------|---------|-------|-------|
+| [Cake .NET Tool]                 | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> |
+| [Cake Frosting]                  | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> |
+| [Cake runner for .NET Framework] | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:orange"></i> [[2]](#footnote2) | <i class="fa fa-check" style="color:orange"></i> [[2]](#footnote2) |
+| [Cake runner for .NET Core]      | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> | <i class="fa fa-check" style="color:green"></i> |
+
+<a id="footnote2"></a>
+[2]: Requiring Mono 5.0.12 or newer
+
+### Supported platforms
+
+The following table shows the supported platforms for each runner.
+
+:::{.alert .alert-info}
+Note that the platform under which a build is running doesn't limit its build capabilities.
+It's absolutely possible to build a .NET Core application with Cake running on .NET Framework or vice-versa.
```

Comment:
```suggestion
It's absolutely possible to build a .NET Framework application with Cake running on .NET 5 / .NET Core or vice-versa.
```

I inverted the statement because I think it's a more common scenario to use the Cake.Tool runner to build legacy .NET Framework apps, than to use the old .NET Framework runner to build .NET Core apps

Reply:
This is copied from the docs page, so in that case needs to be fixed there too.

✔

Reply:
I'll send a PR

Reply:
PR https://github.com/cake-build/website/pull/1558